src/assets/data/defaultConfig.ts

```diff
@@ -184,6 +184,8 @@ export const DefaultNumberOfCompletionLogsToSend = 10
 
 // AUDIO TASK
 
+export const DefaultNumberofAudioAttempts = 15
```

Comment:
Is this the MAX attempts allowed ? If yes then should named appropriately

Reply:
Yes. I'll fix it.